regression_test.py

```diff
@@ -269,6 +269,18 @@ def discover_scenario_tests(regexp, suite):
                     WidelandsTestCase(test_script,
                         scenario=wlmap, script=test_script))
 
+def discover_game_template_tests(regexp, suite):
+    """Add all tests using --new_game_from_template to the 'suite'."""
+    for templ in sorted(glob(os.path.join("test", "templates", "test*.wgt"))):
+        test_script = templ[:-3] + 'lua'
+        if not os.path.isfile(templ) or not os.path.isfile(test_script):
+            continue
```

Comment:
Maybe having a template but no test file should at least print a warning.

Reply:
I check it in the coverage test, because without a script the test won't end. Is that OK for you?